core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -3344,6 +3344,9 @@ public int compare(org.apache.carbondata.format.ColumnSchema o1,
     // for setting long string columns
     if (!longStringColumnsString.isEmpty()) {
       String[] inputColumns = longStringColumnsString.split(",");
+      for (int i = 0; i < inputColumns.length; i++) {
```

Comment:
instead of forloop, you can use `Arrays.stream(inputColumns).map(longColumn => longColumn.trim().toLowerCase()).toArray(String[]::new)` in a more functional manner

Reply:
Done